moj-node/server/services/hubContent.js

```diff
@@ -11,6 +11,11 @@ module.exports = function createHubContentService({
     }
 
     const content = await contentRepository.contentFor(id);
+
+    const suggestedContent = await contentRepository.suggestedContentFor({
```

Comment:
Don't think you need this call here?

Reply:
since you make the same call in the media endpoint

Reply:
Being passed in to a default content item as the ticket said all content items

Reply:
Ahh yes, I'll change 👍

Reply:
right so the remove the extra call in the media fn?